src/Functions/FunctionsMultiStringSearch.h

```diff
@@ -61,7 +61,7 @@ class FunctionsMultiStringSearch : public IFunction
     size_t getNumberOfArguments() const override { return 2; }
     bool useDefaultImplementationForConstants() const override { return true; }
     bool isSuitableForShortCircuitArgumentsExecution(const DataTypesWithConstInfo & /*arguments*/) const override { return true; }
-    ColumnNumbers getArgumentsThatAreAlwaysConstant() const override { return {1}; }
+    ColumnNumbers getArgumentsThatAreAlwaysConstant() const override { return {}; }
```

Comment:
Now we can just remove this line, because this method has default implementation that returns {}
```suggestion
```